### PR TITLE
oracle db added

### PR DESCRIPTION
**Pull request by rohitgodspeed**

### CODE REVIEW

**Review by yaswanth-godspeed — APPROVED** [review]

File: plugins/oracledb-as-datasource/package-lock.json
Comment (siddarthvader) [inline plugins/oracledb-as-datasource/package-lock.json:1]: do not commit this

**Review by siddarthvader — CHANGES_REQUESTED** [review]
@rohitgodspeed please add a README.MD for the instruction to use this plugin,

**Review by siddarthvader — APPROVED** [review]
LGTM

**Comment by siddarthvader** [comment]
@rohitgodspeed also update the readme file of main repo to include that oracle plugin is now supported.

**Pull request closed without merging**